### DISCUSSION TITLE
Is it possible to avoid an error using load fuction on a non-existent file?

### DISCUSSION THREAD

**Discussion by klimenkov** [mikefarah/yq#1571] [Q&A]
I have a file path stored in yaml field and I want to know if I could load the file or not, e.g:
`echo 'null' | yq 'load("non-existing.yaml")'`
gives me an error:
`Error: Failed to load non-existing.yaml: open non-existing.yaml: no such file or directory`
Are there options to check if the file exists or to make the `load` function return a value (so it could be used with an alternative operator `//`) instead of throwing an error?

**Comment by mikefarah** [comment]
Not at the moment sorry - feel free to raise an enhancement request. It would have to be backwards compatible so maybe something like `load_safe(x)` or `load_or_not()` ?
